src/ndarray/ndarray.cc

```diff
@@ -1711,7 +1711,10 @@ bool NDArray::Load(dmlc::Stream *strm) {
   // load shape
   mxnet::TShape shape;
   if (!shape.Load(strm)) return false;
-  if (shape.ndim() == 0) {
+  if (Imperative::Get()->is_np_comp()) {
```

Comment:
Actually, it should be `if (!Imperative::Get()->is_np_comp()) {`.

Reply:
My bad...fixed!